playbooks/roles/cluster/tasks/_remove_dns_records.yml

```diff
@@ -0,0 +1,25 @@
+- route53: 
+    aws_access_key: "{{ fheng_access_key_id }}"
+    aws_secret_key: "{{ fheng_secret_access_key }}"
+    record: "{{ record }}"
+    region: "{{ aws_cluster_name }}"
+    state: get
+    type: CNAME
+    zone: "{{ aws_dns_zone }}"
+  register: get_aws_dns_record
+
+- set_fact:
+    aws_dns_record: "{{ get_aws_dns_record.set }}"
+
+- route53:
+    aws_access_key: "{{ fheng_access_key_id }}"
```

Comment:
I'm not sure we can specifically target the fheng key here, it might be best to use aws_access_key_id and populate that with {{ fheng_access_key_id }} as the input params or something